### PR TITLE
Fix bug where loadWalletBalances saga doesn't get any network info if noPoll is true

### PR DESCRIPTION
**Pull request by liamaharon**
Also re-enables some accidentally disabled tests

### CODE REVIEW

File: src/containers/HubiiApiHoc/tests/saga.test.js [hunk 1]
```diff
@@ -229,6 +229,15 @@ describe('hubiiApi saga', () => {
     beforeEach(() => {
       saga = testSaga(loadWalletBalances, { address }, currentNetworkMock.walletApiEndpoint);
     });
+
+    it('should select currentNetwork inside the saga network if noPoll is true', () => {
```
Comment (katat) [inline src/containers/HubiiApiHoc/tests/saga.test.js]: Just for my information, why the network selection depends on this noPoll flag?

Comment (liamaharon) [reply src/containers/HubiiApiHoc/tests/saga.test.js]: Usually, it's passed in via the orchestrator (see ln 135). Doesn't really make a difference

File: src/containers/HubiiApiHoc/saga.js [hunk 2]
```diff
@@ -39,16 +39,16 @@ import {
 } from './actions';
 
 
-export function* loadWalletBalances({ address, noPoll }, network) {
+export function* loadWalletBalances({ address, noPoll }, _network) {
+  let network = _network;
```
Comment (katat) [inline src/containers/HubiiApiHoc/saga.js]: would it make sense to replace this line with `let network = _network || yield  select(makeSelectCurrentNetwork())` and remove the line `if (noPoll) network = yield select(makeSelectCurrentNetwork());`?

Comment (liamaharon) [reply src/containers/HubiiApiHoc/saga.js]: Good idea, will make the change

**Pull request merged**